datafusion/sql/src/unparser/dialect.rs

```diff
@@ -168,6 +173,17 @@ pub enum DateFieldExtractStyle {
     Strftime,
 }
 
+/// `CharacterLengthStyle` to use for unparsing
+///
+/// Different DBMSs uses different names for function calculating the number of characters in the string
+/// `Length` style uses length(x)
+/// `SQLStandard` style uses character_length(x)
+#[derive(Clone, Copy, PartialEq)]
+pub enum CharacterLengthStyle {
+    Length,
+    SQLStandard,
+}
```

Comment:
```suggestion
pub enum CharacterLengthExtractStyle {
    Length,
    CharacterLength,
}
```

Let's align them to the name of the function they're associated with, similar to `DateFieldExtractStyle`.

I think we could also rename this enum to align with that similar naming, like `CharacterLengthExtractStyle`.

Reply:
That makes sense, let me update it to `CharacterLength `